app/controllers/application_controller.rb

```diff
@@ -1790,6 +1790,15 @@ def task_supported?(typ)
     end
 
     vms = VmOrTemplate.where(:id => vm_ids)
+    if typ == "migrate"
+      if vms.map(&:ext_management_system).uniq.compact.any? do |ems|
+        ems.respond_to?(:supports_migrate_for_all?) && !ems.supports_migrate_for_all?(vms)
```

Comment:
ok now I see this logic could be a bit improved also. We are passing *all* selected vms to each individual `ems` to check whether they can be migrated together but IMO each `ems` should have only say about his own subset of vms. Of course this accidentally works since the RHV implementation checks for the same cluster id which across providers will never hold, but that I don't believe that is the robustness we want :-) Will post a fix.